lib/matplotlib/tests/test_offsetbox.py

```diff
@@ -99,3 +99,137 @@ def test_offsetbox_loc_codes():
         anchored_box = AnchoredOffsetbox(loc=code, child=da)
         ax.add_artist(anchored_box)
     fig.canvas.draw()
+
+
+@image_comparison(baseline_images=['rasterized_patch'],
+                  extensions=['pdf'], tol=0.01)
```

Comment:
You can list multiple extensions `extensions=['pdf', 'png']` to reduce the number of tests you need to add.